images/warp10/s6/cont-init.d/40-generate-secrets

```diff
@@ -10,10 +10,10 @@ WARP10_CONFIG_DIR="$WARP10_DATA_DIR/conf"
 INITIAL_TOKENS="$WARP10_CONFIG_DIR/initial.token"
 
 if [ ! -f "$INITIAL_TOKENS" ]; then
-    CONFIG_FILES="$(find ${WARP10_CONFIG_DIR} -not -path "*/\.*" -name "*.conf" | sort | tr '\n' ' ' 2> /dev/null)"
+    CONFIG_FILES="$(find ${d} -not -path "*/\.*" -name "*.conf" | sort | tr '\n' ' ' 2> /dev/null)"
```

Comment:
What is `${d}` here?

Reply:
A bug :) It should have not been changed. I've got to check and see why it didn't break the CI as it shouldn't be working if this broken